src/cfclient/ui/tabs/FlightTab.py

```diff
@@ -205,6 +207,9 @@ def __init__(self, tabWidget, helper, *args):
             lambda: self._flight_command(CommanderAction.DOWN)
         )
 
+        self.joystickReader = JoystickReader()
```

Comment:
Does this need to be a member variable?

Reply:
nope it doesn't indeed! wanted to see if I could do more updates of this for the the flight tab, but that requires some intergrate changes to the cfclient. For another time!